src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final int CHARACTERISTIC_POSITION_IN_ARRAY = 0;
+    public static final int VALUE_POSITION_IN_ARRAY = 1;
```

Comment:
Variable names should be consistent with the task description
We have 'operation' and 'amount' and do not have 'CHARACTERISTIC' or 'VALUE'

![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/d2de88e8-954f-4242-a58d-4078268ea43b)